From 3fd21a6b776e0f874e0e14d9943ac2b06bcc4aad Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michel=20D=C3=A4nzer?= <michel.daenzer@amd.com>
Date: Fri, 28 Jun 2019 11:07:39 +0200
Subject: [PATCH] targets/opencl: Add clangASTMatchers library as dependency

Fixes link failure since clang r364424 "[clang/DIVar] Emit the flag for
params that have unmodified value", clangCodeGen depends on
clangASTMatchers now.

Reviewed-by: Dylan Baker <dylan@pnwbakers.com>
---
 src/gallium/targets/opencl/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gallium/targets/opencl/meson.build b/src/gallium/targets/opencl/meson.build
index 317ad8dab4a..650a2054cfb 100644
--- src/gallium/targets/opencl/meson.build
+++ src/gallium/targets/opencl/meson.build
@@ -51,6 +51,7 @@ libopencl = shared_library(
     cpp.find_library('clangSema', dirs : llvm_libdir),
     cpp.find_library('clangAnalysis', dirs : llvm_libdir),
     cpp.find_library('clangAST', dirs : llvm_libdir),
+    cpp.find_library('clangASTMatchers', dirs : llvm_libdir),
     cpp.find_library('clangEdit', dirs : llvm_libdir),
     cpp.find_library('clangLex', dirs : llvm_libdir),
     cpp.find_library('clangBasic', dirs : llvm_libdir),
